### PR TITLE
Update GH issue templates

### PR DESCRIPTION
**Pull request by morganchen12**
* Added an `Installation method` field to the issue template and instructed users to delete the guidelines before filing their issues.
* Added a separate feature request template.

#no-changelog

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### [READ] Guidelines
+
+When filing a feature request please make sure the issue title starts with "FR:".
```
Comment (samtstern) [inline .github/ISSUE_TEMPLATE/feature_request.md]: By the way you can tell GitHub to auto-label issues with a certain template.  So you could give this one `type: feature request` automatically.

See:
https://raw.githubusercontent.com/google/ExoPlayer/release-v2/.github/ISSUE_TEMPLATE/feature_request.md

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+### Step 0: Are you in the right place?
```
Comment (samtstern) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Do you want to add the metadata to the top of this file?

Example:
https://raw.githubusercontent.com/google/ExoPlayer/release-v2/.github/ISSUE_TEMPLATE/bug.md

**Comment by samtstern** [comment]
@morganchen12 if this repo uses template validation by the oss-bot make sure to read the section here on multiple templates:
https://github.com/firebase/oss-bot#template-matching

**Comment by samtstern** [comment]
@morganchen12 don't forget the other metadata (the one in the md comment)

**Comment by morganchen12** [comment]
@samtstern wasn't sure if the oss-bot template location metadata should go at the top of the template file. Please check that I did it right

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Issue template for a bug report.
+title: ''
```
Comment (samtstern) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think you can leave out anything that's `''`

**Comment by samtstern** [comment]
@morganchen12 yeah it can go anywhere but I find it least likely to be deleted at the top.

**Comment by morganchen12** [comment]
oss-bot CL at cr/291028440

**Pull request merged**